CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Optimizely Python SDK Changelog
 
+## 3.6.0-beta
+September 21st, 2020
+
+### New Features:
+* Version targeting using semantic version syntax [#293](https://github.com/optimizely/python-sdk/pull/293)
```

Comment:
nit. Period after last word in all of these updates. So after syntax, accessor and pypy.